firmware/applications/SignBuddy/nucleo_board.h

```diff
@@ -0,0 +1,53 @@
+#ifndef NUCLEO_BOARD_H
+#define NUCLEO_BOARD_H
+
+#include "stm32l4xx_ll_bus.h"
+#include "stm32l4xx_ll_cortex.h"
+#include "stm32l4xx_ll_dma.h"
+#include "stm32l4xx_ll_exti.h"
+#include "stm32l4xx_ll_gpio.h"
+#include "stm32l4xx_ll_lpuart.h"
+#include "stm32l4xx_ll_pwr.h"
+#include "stm32l4xx_ll_rcc.h"
+#include "stm32l4xx_ll_system.h"
+#include "stm32l4xx_ll_tim.h"
+#include "stm32l4xx_ll_usart.h"
+#include "stm32l4xx_ll_utils.h"
+
+#define SYSCLK_FREQ    64000000
+
+// TODO: change LED pins to be correct for nucleo
+/* LED */
+#define LED1_PORT               GPIOB
+#define LED1_PIN                LL_GPIO_PIN_2
+
+#define GPIO_LED_CLK_EN()       LL_AHB2_GRP1_EnableClock(LL_AHB2_GRP1_PERIPH_GPIOB)
+
+/* System Time */
+#define SYSTEM_TIME_TIMER       TIM2
+#define SYSTEM_TIME_CLK_EN()    LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_TIM2)
+
+/* Bluetooth UART */
+#define LPUART_PORT             GPIOC
+#define LPUART_RX_PIN           LL_GPIO_PIN_0
+#define LPUART_TX_PIN           LL_GPIO_PIN_1
+#define GPIO_LPUART_CLK_EN()    LL_AHB2_GRP1_EnableClock(LL_AHB2_GRP1_PERIPH_GPIOC)
+
+#define LPUART_CLK_EN()         LL_APB1_GRP2_EnableClock(LL_APB1_GRP2_PERIPH_LPUART1)
+
+/* UART */
+#define UART                          USART3
+#define UART_IRQn                     USART3_IRQn
+#define UART_IRQHandler               USART3_IRQHandler
+
+#define UART_RX_PIN                   LL_GPIO_PIN_8
+#define UART_TX_PIN                   LL_GPIO_PIN_9
+#define UART_GPIO_PORT                GPIOD
+#define UART_GPIO_AF                  LL_GPIO_AF_7
+
+#define UART_BAUDRATE                 115200
+
+#define UART_CLK_EN()                 LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_USART3)
+#define GPIO_UART_CLK_EN()            LL_AHB1_GRP1_EnableClock(LL_AHB1_GRP1_PERIPH_GPIOD)
```

Comment:
rename UART used for logging to LOGGER_UART or something like that and the bluetooth one something similar. since the logger uart is routed to the st-link we should use different hardware modules, but combine them into one `uart.c` module

Reply:
i changed one of them.. someone do the other pls